kraken-build/src/kraken/std/python/buildsystem/uv.py

```diff
@@ -244,9 +254,9 @@ def build_v2(self, settings: PythonSettings, output_directory: Path) -> list[Pat
             if shutil.which("uv") != self.uv_bin:
                 env["PATH"] = str(Path(self.uv_bin).parent) + os.pathsep + env["PATH"]
 
-            # We can't pass the --index-url and --extra-index-url options to UV via the pyproject-build CLI,
+            # We can't pass the --default-index and --index options to UV via the pyproject-build CLI,
```

Comment:
Not sure what this comment is about. Is it up to date ?

Reply:
Here I'm just updating the CLI options as `--index-url` and `--extra-index-url` are marked as deprecated.
The comment is currently still valid because `pyproject-build` can't forward arguments when calling `uv`. As such the configuration is converted into a set of environment variables.

I plan to rework this part in a follow up MR. `uv build` works even when no `build-system` is specified in the `pyproject.toml`.